src/03-deals/deals.apib

```diff
@@ -157,6 +157,11 @@ Get details for a single deal.
             + updated_at: `2017-05-09T11:30:58+00:00` (string)
             + `web_url`: `https://app.teamleader.eu/sale_detail.php?id=f6871b06-6513-4750-b5e6-ff3503b5a029` (string)
             + custom_fields (array[CustomField])
+            + lost_reason (object)
```

Comment:
PP: place the `lost_reason` _before_ meta data properties (`created_at`, `updated_at` and `web_url`).